packages/stacking/README.md

```diff
@@ -0,0 +1,94 @@
+# @stacks/stacking [![npm](https://img.shields.io/npm/v/@stacks/stacking?color=red)](https://www.npmjs.com/package/@stacks/stacking)
+Library for PoX Stacking.
+
+## Installation
+
+```
+npm install @stacks/stacking
+```
+
+## Initialization
+```typescript
+const { StacksTestnet } = require('@stacks/network');
+const { Stacker } = require('@stacks/stacking');
+
+const address = 'ST3XKKN4RPV69NN1PHFDNX3TYKXT7XPC4N8KC1ARH';
+const network = new StacksTestnet();
+const stacker = new Stacker(address, network);
+```
+
+## Check stacking eligibility
+
+```typescript  
+const stackingEligibility = await stacker.canLockStx({poxAddress, cycles});
```

Comment:
Should this be rather `isEligible`? If I recall correctly, the method doesn't check for balances of the STX account. So this could be true but locking wouldn't be enabled

Reply:
The implementation of `canLockStx()` will fetch the account balance and use that as an input to the contract function call, so it does take that into account.